component/scriptsPermissions.jsonnet

```diff
@@ -0,0 +1,63 @@
+/*
+    This Job ensures smooth upgrade of Stackgres from version > 1.5.
+    It solves issues with breaking changes introduced in helm charts, as well as egg-chicken (certs, issuers, secrets) issues during upgrade.
+*/
+
+local kap = import 'lib/kapitan.libjsonnet';
+local inv = kap.inventory();
+local kube = import 'lib/kube.libjsonnet';
+local params = inv.parameters.stackgres_operator;
+local instance = inv.parameters._instance;
+
+
+local rolename = 'stackgres-init-additional-permissions';
+
+local role = kube.Role(rolename) {
+  metadata+: {
+    annotations: {
+      'argocd.argoproj.io/hook': 'PreSync',
+    },
+  },
+  rules: [
+    {
+      apiGroups: [ '' ],
+      resources: [ 'secrets' ],
+      verbs: [ 'get', 'list', 'watch', 'create', 'update', 'delete' ],
+    },
+    {
+
+      apiGroups: [ '' ],
+      resources: [ 'clusterrolebindings' ],
+      verbs: [ 'delete' ],
+    },
+    {
+
+      apiGroups: [ 'cert-manager.io' ],
+      resources: [ 'certificates', 'issuers' ],
+      verbs: [ 'delete' ],
+    },
+  ],
+};
+
+local roleBinding = kube.RoleBinding(rolename + '-rolebinding') {
```

Comment:
No need to suffix `rolebinding` here. It's already clear that it's a rolebinding